### PR TITLE
modified areaCalculator application

### PR DESCRIPTION
**Pull request by marekk1992**

### CODE REVIEW

File: src/areaCalculator/Main.java [hunk 1]
```diff
@@ -1,8 +1,37 @@
 package areaCalculator;
```
Comment (RobertasVainickas) [inline src/areaCalculator/Main.java]: Why everything is in single file?

File: src/areaCalculator/Main.java [hunk 2]
```diff
@@ -1,8 +1,37 @@
...
 
+class Rectangle {
+    private double lengthA;
```
Comment (RobertasVainickas) [inline src/areaCalculator/Main.java]: What is A? What is B? 
I think names could be more accurate. 
Remember how you call those two things mathematically

File: src/areaCalculator/Main.java [hunk 3]
```diff
@@ -1,8 +1,37 @@
...
+                + rectangle.getLengthB() + ") = " + area);
+    }
+}
+
 public class Main {
 
     public static void main(String[] args) {
-
+        Rectangle rectangle = new Rectangle(15.2, 22.3);
+        AreaCalculator area = new AreaCalculator();
```
Comment (RobertasVainickas) [inline src/areaCalculator/Main.java]: From the name of the variable I would assume that it stores area, but it actually stores area calculator.

**Review by RobertasVainickas — CHANGES_REQUESTED** [review]

File: src/areaCalculator/Main.java (hunk 1 above)
Comment (marekk1992) [reply src/areaCalculator/Main.java]: classes separated

File: src/areaCalculator/Main.java (hunk 2 above)
Comment (marekk1992) [reply src/areaCalculator/Main.java]: Changed to length and width

File: src/areaCalculator/Main.java (hunk 3 above)
Comment (marekk1992) [reply src/areaCalculator/Main.java]: fixed

File: src/areaCalculator/Main.java [hunk 4]
```diff
@@ -3,6 +3,8 @@
 public class Main {
 
     public static void main(String[] args) {
-
+        AreaCalculator myCalculator = new AreaCalculator();
```
Comment (RobertasVainickas) [inline src/areaCalculator/Main.java:6]: Variable could be named simply calculator, prefix "my" does not add any real value here. 
Anyway, it is optional, you can fix it with other PR if you want.

**Review by RobertasVainickas — APPROVED** [review]

**Pull request merged**